sklearn/metrics/_classification.py

```diff
@@ -2528,9 +2528,11 @@ def log_loss(
         ordered alphabetically, as done by
         :class:`preprocessing.LabelBinarizer`.
 
-    eps : float, default=1e-15
+    eps : float or "auto", default="auto" or 1e-15
         Log loss is undefined for p=0 or p=1, so probabilities are
         clipped to max(eps, min(1 - eps, p)).
+        Default value will be the epsilon of the dtype of y_pred if it is
+        a numpy array or 1e-15 if a non-numpy input is passed.
 
```

Comment:
This will need a `..versionadded:: 1.2` directive.

Reply:
Can you give an example of how this would look?

Reply:
See for example:
https://github.com/scikit-learn/scikit-learn/blob/db8d67c6a10ef37bf58088e414bbe8a7f7160c60/sklearn/kernel_approximation.py#L94-L98